doc/classes/ProjectSettings.xml

```diff
@@ -2364,6 +2364,136 @@
+			[b]Note:[/b] Enabling this setting can cause certain shapes to be culled from the results entirely, but you will get at least one intersection per body.
+			[b]Note:[/b] This will only remove edge collisions internal to a single body, meaning edges between separate bodies can still cause ghost collisions.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/allow_sleep" type="bool" setter="" getter="" default="true">
+			Whether [RigidBody3D] is allowed to go to sleep.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/areas_detect_static_bodies" type="bool" setter="" getter="" default="false">
+			Whether or not [Area3D] is able to detect overlaps with static physics bodies, such as [StaticBody3D].
+			[b]Note:[/b] Enabling this setting can come at a heavy CPU and memory cost if you allow many/large [Area3D] to overlap with complex static geometry, such as [ConcavePolygonShape3D] or [HeightMapShape3D]. It is strongly recommended that you set up your collision layers and masks in such a way that only a few small [Area3D] can detect static bodies.
+			[b]Note:[/b] This also applies to overlaps with [RigidBody3D] frozen with [constant RigidBody3D.FREEZE_MODE_STATIC].
+			[b]Note:[/b] This is not needed to detect overlaps with [AnimatableBody3D], which is a kinematic body.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/baumgarte_stabilization_factor" type="float" setter="" getter="" default="0.2">
+			How much of the position error of a [RigidBody3D] to fix during a physics step, where 0 is none and 1 is the full amount, which affect things like how quickly bodies depenetrate.
+			[b]Note:[/b] Setting this too high can result in instability for [RigidBody3D].
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/body_pair_contact_cache_angle_threshold" type="float" setter="" getter="" default="0.0349066">
+			The maximum relative angle by which a body pair can move and still reuse the collision results from the previous physics step.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/body_pair_contact_cache_distance_threshold" type="float" setter="" getter="" default="0.001">
+			The maximum relative distance by which a body pair can move and still reuse the collision results from the previous physics step.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/body_pair_contact_cache_enabled" type="bool" setter="" getter="" default="true">
+			Whether the body pair contact cache is enabled, which removes the need for potentially expensive collision detection when the relative orientation between two bodies hasn't changed much.
+		</member>
+		<member name="physics/jolt_physics_3d/simulation/bounce_velocity_threshold" type="float" setter="" getter="" default="1.0">
+			The minimum velocity needed before a collision can be elastic.
```

Comment:
What does an elastic collision mean in this context? Is it specially handled by Jolt, or does it do something specific in Godot?

Reply:
It's a Jolt feature. An elastic collision is a collision with restitution != 0, basically it determines the velocity that is needed before the restitution calculations will kick in. If the velocity is lower than this value, the simulation will treat the collision as having restitution = 0 regardless of what the user specified.

Reply:
Maybe it sounds silly, but seeing as how Godot refers to restitution as bounce in its physics materials (hence the setting name), would it be wrong to just say this instead?

```suggestion
			The minimum velocity needed before a collision can be bouncy, in meters per second.
```